index.bs

```diff
@@ -2211,17 +2217,16 @@ Reopening a Decision When Presented With New Information</h3>
 [=Chair Decision=] and [=Group Decision=] Appeals</h3>
 
 	When group participants believe that their concerns are not being duly considered by the group or the [=Chair=],
-	they <em class="rfc2119">may</em> ask the <a href="#def-Director">Director</a>
-	(for representatives of a Member organization, via their Advisory Committee representative)
-	to confirm or deny the decision.
+	they <em class="rfc2119">may</em> ask that the decision be confirmed or denied
+	(for representatives of a Member organization, via their Advisory Committee representative).
 	This is a <dfn export id="wg-decision-appeal">Group Decision Appeal</dfn>
 	or a <dfn export id="chair-decision-appeal">Chair Decision Appeal</dfn>.
 	The participants <em class="rfc2119">should</em> also make their requests known
 	to the <a href="#TeamContact">Team Contact</a>.
-	The Team Contact <em class="rfc2119">must</em> inform the Director
-	when a group participant has raised concerns about due process.
+	The Team Contact <em class="rfc2119">must</em> inform the [=CEO=]
+	when a group participant has also raised concerns about due process.
```

Comment:
why "also"?

Reply:
Because the participant might have raised an FO, in which case the CEO doesn't need to be informed, but if they raised concerns about due process specifically, the CEO must be informed. (The “also” is clarifying that this is a distinct additional requirement for this subcase.)